components/cardsTable/index.tsx

```diff
@@ -31,7 +33,7 @@ export const CardsTable = ({ cards, handleDelete }) => {
                 className="place-items-end min-w-max h-50 rounded-md text-sky-500 rounded-md border-grey"
```

Comment:
`h-50` does not exist in tailwind. Also, you have duplicated `rounded-md`. You don't need `place-items-end` too.